src/main/java/com/indico/request/GraphQLRequest.java

```diff
@@ -0,0 +1,68 @@
+package com.indico.request;
+
+import com.apollographql.apollo.exception.ApolloException;
+import com.indico.IndicoClient;
+import com.indico.RestRequest;
+import java.io.IOException;
+import okhttp3.MediaType;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class GraphQLRequest implements RestRequest<JSONObject> {
+
+    private final IndicoClient client;
+    private String query;
+    private String operationName;
+    private JSONObject variables;
+
+    public GraphQLRequest(IndicoClient client) {
+        this.client = client;
+    }
+
+    public GraphQLRequest query(String query) {
+        this.query = query;
+        return this;
+    }
+
+    public GraphQLRequest operationName(String operationName) {
```

Comment:
Operation name can be removed it doesn't do anything sent separately.

Reply:
Fixed.